### ISSUE TITLE
Can not open maps on desktop, app crashes

### ISSUE THREAD

**Issue by eekkelund** [rinigus/pure-maps#333]
On desktop when trying to open a map app crashes with error:
`qmlscene: symbol lookup error: /usr/lib/qml/MapboxMap/libqmlmapboxglplugin.so: undefined symbol: ubidi_open_63`
If no Mapbox API key inserted and Mapbox used as default map, app works until choosing map provider eg. HSL. Then immediately crashes with above error.

Tried with `use_icu` and without `use_icu` CONFIG https://github.com/rinigus/pkg-mapbox-gl-native/blob/qt-regular/mapbox-gl-native-lib.pro#L5

Self built [mapbox-gl-qml](https://github.com/rinigus/mapbox-gl-qml), master branch
Self built [pkg-mapbox-gl-native](https://github.com/rinigus/pkg-mapbox-gl-native) qt-regular branch
Using qtquickcontrols 2 and Qt version 5.12.5.

**Comment by rinigus** [comment]
Sorry for delayed reply.

ICU should be linked via mapbox gl native. 

If you are installing on QEMU regular Linux, then I would suggest to use

* https://github.com/rinigus/pkg-mapbox-gl-native/tree/qt-regular That's a branch which is used on my Debian package and PostmarketOS (to my knowledge). Try to use it without use_icu (see PRO file)

* then recompile fresh Mapbox GL QML. 

I don't see why would ICU be referenced from QML module though

**Comment by rinigus** [comment]
PS: on my system, grep icu on /usr/lib/qml/MapboxMap/libqmlmapboxglplugin.so has no hits

**Comment by eekkelund** [comment]
Thanks for your help.
I built whole system again adding `-icu` support to Qt.
Otherwise I used same branches and configs as described in first comment.
In addition I noticed that `bash python3-xml python3-lxml python3-numbers` are runtime dependencies that I didn't have installed. 
Pure-maps with online maps are working now!

Thank you for these great applications! I use them multiple times a day on my SFOS device! :smile: 

As a side comment, now I am trying to build osmscout-server. I don't have QtWidgets installed so I will try to fork osmscout-server. For a same reason I use qmlscene and not qmlrunner, as I don't have those QtWidgets.
